evergreen.yml

```diff
@@ -320,199 +317,197 @@ functions:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - dry-run-workloads
-      - --workload
-      - ./src/genny/src/workloads/docs/SelfTestActor.yml
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - dry-run-workloads
+          - --workload
+          - ./src/genny/src/workloads/docs/SelfTestActor.yml
 
   ##
   # Runs tests via ctest.
   # Requires f_compile to have been run first.
   ##
   f_cmake_test:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - cmake-test
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - cmake-test
 
   ##
   # Runs tests via ctest, ignoring flaky tests (see EVG-19909).
   # Requires f_compile to have been run first.
   # Feel free to update regex_clude per the latest test results
   ##
   f_cmake_test_ignore_flaky:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - cmake-test
-      - --regex-exclude
-      - "IgnoredOnMacOs"
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - cmake-test
+          - --regex-exclude
+          - "IgnoredOnMacOs"
 
   ##
   # Runs benchmarks via ctest.
   # Requires f_compile to have been run first.
   ##
   f_benchmark_test:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - benchmark-test
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - benchmark-test
 
   ##
   # Runs tests for the create_new_actor script.
   ##
   f_run_create_new_actor_script:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - create-new-actor
-      - SelfTestActor
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - create-new-actor
+          - SelfTestActor
 
   ##
   # Runs python unit tests.
   ##
   f_python_test:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      # TODO: rename to pytest
-      - self-test
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          # TODO: rename to pytest
+          - self-test
 
   ##
   # Runs the YAML linter for Genny workloads
   ##
   f_lint_workloads:
-  - command: subprocess.exec
-    params:
-      binary: ./src/genny/run-genny
-      args:
-      - -v
-      - lint-yaml
-
+    - command: subprocess.exec
+      params:
+        binary: ./src/genny/run-genny
+        args:
+          - -v
+          - lint-yaml
+          - --format
```

Comment:
Did you add this? I was hoping / assuming that all these were just whitespace linting changes.

Reply:
yes, I did it. Without this option, this command does not check the format of the yaml files.

Reply:
:+1: